release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -1217,10 +1217,22 @@ OpenShift SDN network plugin was deprecated in 4.15 and 4.16. With this release,
 [id="ocp-4-17-cloud-cred-operator-bug-fixes_{context}"]
 ==== Cloud Credential Operator
 
+* Previously, secrets in the cluster were fetched in a single call.
+When there were a large number of secrets, this caused the API to time out.
+With this release, the Cloud Credential Operator fetches secrets in batches limited to 100 secrets. This change prevents timeouts when there are large number of secrets in the cluster. (link:https://issues.redhat.com/browse/OCPBUGS-41233[*OCPBUGS-41233*])
+
+* Previously, the Cloud Credential Operator reported an error when the `awsSTSIAMRoleARN` role was not present on a cluster that uses manual mode with AWS STS. With this release, the Cloud Credential Operator is updated to no longer report this as an error. (link:https://issues.redhat.com/browse/OCPBUGS-33566[*OCPBUGS-33566*])
+
+* Previously, when checking whether passthrough permissions are sufficient, the Cloud Credential Operator sometimes received a response from the Google Cloud Platform API that a permission is invalid for a project. This response caused the Operator to become degraded and installation to fail. With this release, the Operator is updated to handle this error gracefully. (link:https://issues.redhat.com/browse/OCPBUGS-36140[*OCPBUGS-36140*])
+
 [discrete]
 [id="ocp-4-17-cluster-version-operator-bug-fixes_{context}"]
 ==== Cluster Version Operator
 
+* Previously, on rare conditions, a race condition between Go routines caused the Cluster Version Operator (CVO) to panic after the CVO started. With this release, the Go routines synchronization is improved and the issue is resolved. (link:https://issues.redhat.com/browse/OCPBUGS-32678[*OCPBUGS-32678*])
+
+* Previously, standalone clusters running 4.17.0-ec.0 with the default feature set could not find and download release signatures. With this release, the ability to download release signatures is restored and the issue is resolved. (link:https://issues.redhat.com/browse/OCPBUGS-35236[*OCPBUGS-35236*])
```

Comment:
```suggestion
* Previously, standalone clusters running the `4.17.0-ec.0` engineering candidate with the default feature set could not find and download release signatures. With this release, the ability to download release signatures is restored and the issue is resolved. (link:https://issues.redhat.com/browse/OCPBUGS-35236[*OCPBUGS-35236*])
```
So, to my knowledge, an `ec` build is not a public release. Should this even have a release note? The release itself will be an `rc`. (Maybe I am wrong, but it can't hurt to check.)

Reply:
Looks like a release note is not necessary here.